testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -311,6 +303,30 @@ def test_check_header_title_multiframe(self):
                     got_title += 1
                     assert got_title <= 1, "There should be only one TITLE."
 
+    @pytest.mark.parametrize("startframe,maxframes",
+                             [(0, 12), (9997, 12)])
+    def test_check_MODEL_multiframe(self, universe2, outfile, startframe, maxframes):
+        """Check whether MODEL number is in the right column (Issue #1950)"""
+        u = universe2
+        protein = u.select_atoms("protein and name CA")
+        with mda.Writer(outfile, multiframe=True, start=startframe) as pdb:
+            for ts in u.trajectory[:maxframes]:
+                pdb.write(protein)
+
+        def get_MODEL_lines(filename):
+            with open(filename) as pdb:
+                for line in pdb:
+                    if line.startswith("MODEL"):
+                        yield line
+
+        MODEL_lines = list(get_MODEL_lines(outfile))
+
+        assert len(MODEL_lines) == maxframes
+        for model, line in enumerate(MODEL_lines, start=startframe+1):
+            # test that only the right-most 4 digits are stored (rest must be space)
+            # line[10:14] == '9999' or '   1'
+            assert line[5:14] == "{0:>9d}".format(int(str(model)[-4:]))
```

Comment:
This tests may be better if it tested the integer rather than the string. That would make the implementation of the test different from the implementation of the feature.

Reply:
I can add it. I wanted to make sure that the whitespace is also as expected.

Reply:
added in 8dddab15954fa09002c620bde34b3dcd017faa87